src/_playground/documentation/Properties/Properties.js

```diff
@@ -16,13 +16,27 @@ export const Properties = ({ sourceModulePath }) => {
             <h2>Properties</h2>
             {componentNames.map((name, index) => {
                 const component = sourceModule[name];
+                const subcomponentNames = Object.keys(component)
+                    .sort()
+                    .filter(subName => component[subName] && component[subName].displayName);
                 return (
                     <React.Fragment key={index}>
-                        <h3 className='header'>{name}</h3>
                         <PropertyTable
                             defaultProps={component.defaultProps}
                             propDescriptions={component.propDescriptions}
-                            propTypes={component.propTypes} />
+                            propTypes={component.propTypes}
+                            title={name} />
+                        {subcomponentNames.map((subName, subIndex) => {
+                            const subcomponent = component[subName];
+                            return (
+                                <PropertyTable
+                                    defaultProps={subcomponent.defaultProps}
+                                    key={subIndex}
+                                    propDescriptions={subcomponent.propDescriptions}
+                                    propTypes={subcomponent.propTypes}
+                                    title={`${name}.${subName}`} />
+                            );
+                        })}
```

Comment:
Added this additional loop to include all sub-components as well.  This still works if the component doesn't have any sub-components.